src/js/util/stringUtil.js

```diff
@@ -0,0 +1,26 @@
+import { Message } from "./Message.js";
+
+const OVER_NUMBER = /(\d+){3,}/;
+
+export function validOverNumber(input) {
+  if (input.match(OVER_NUMBER) !== null) {
+    alert(Message.ILLEGAL_OVER_OPERAND);
+    return false;
+  }
+  return true;
+}
+
+export function validNoNumber(input) {
+  if (input === '0') {
+    return true;
+  }
+  return false;
+}
```

Comment:
```suggestion
export function validNoNumber(input) {
  return input === '0'
}
```
이렇게도 작성하실 수 있을것 같아요!